### PR TITLE
dressup expansion in the afterlife

### PR DESCRIPTION
**Pull request by Retrino**
## About the PR 

expands the after life clothing area to have departmental clothing vendors with all clothing options unlocked in them (april fools, winter, that kind of thing). new subtype of the vending machines called "afterlife" which removes access requirements and pda/radios from the purchase list. rearrange the area a little to not take too much space, kitchen got moved down a tile and the gaming machines got their own little spot next to the clothing area door

specifically, what has been added:
- security, engineering, medical, civilian, and research job clothing vendors
- four boxes containing hat, shoe, clothing, and formal random spawn piles
- sparkles

image
image2

## Why's this needed? 

more afterlife content has always been something requested by players. while this isn't that large of an addition on that front, it's still a lil something extra to do for ghosts. it's also a nice area for people trying to figure out a character look without having to waste a job slot just to see how they might look in that job's clothes, helpful for those who aren't familiar with running debug servers to spawn the items in and such (and a time saver for those that do).

## Changelog 

```changelog
(u)Retrino
(+)The afterlife bar now has an expanded dress-up parlor. 
```

### CODE REVIEW

**Comment by Retrino** [comment]
for some reason my saturation doesnt work ingame but does in screenshots?? weird

i also removed the atm by accident (is this even used?) adding it back in a sec

**Comment by Retrino** [comment]
grr

File: maps/z2.dmm [hunk 1]
```diff
@@ -57845,9 +57936,7 @@
 	},
 /area/centcom)
 "gAK" = (
-/obj/submachine/claw_machine,
-/turf/unsimulated/floor/wood/six,
-/area/afterlife/bar/sanctuary)
+)
```
Comment (Sovexe) [inline maps/z2.dmm:57939]: several sections of the map are missing turfs entirely

**Review by Sovexe — CHANGES_REQUESTED** [review]

File: maps/z2.dmm (hunk 1 above)
Comment (Retrino) [reply maps/z2.dmm:57939]: not sure how the fuck that happened but it should be fixed

**Pull request merged**